public/app/core/components/editors/DashboardPickerByID.tsx

```diff
@@ -10,7 +10,7 @@ import { SelectableValue } from '@grafana/data';
 export interface DashboardPickerItem {
   id: number;
   uid: string;
-  label: string;
+  [key: string]: any;
```

Comment:
maybe `[key: string]: string;` instead?

`any` is about to be forbidden by lint rule...